src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 CLOUD_PROVIDER=${1:-$CLOUD_PROVIDER}
+# Enabling dynamically loaded cloud provider external plugin feature, disabled by default
+ENABLE_CLOUD_PROVIDER_EXTERNAL_PLUGIN=${ENABLE_CLOUD_PROVIDER_EXTERNAL_PLUGIN:-false}
+# caa will try to load cloud provider external plugin from "${CLOUD_PROVIDER_EXTERNAL_PLUGIN_PATH}" in runtime
+# The values of 1) ${CLOUD_PROVIDER}, 2) the filename of the cloud provider external plugin
+# and 3) the provider defined within the external plugin must all match
+# specific env variables for any new cloud provider external plugin must be set and valid
+CLOUD_PROVIDER_EXTERNAL_PLUGIN_PATH=${CLOUD_PROVIDER_EXTERNAL_PLUGIN_PATH:-/run/cloud-api-adaptor/plugins/libvirt.so}
```

Comment:
nit: CLOUD_PROVIDER_EXTERNAL_PLUGIN_PATH should not have a default value.

Reply:
removed, as we have logical in golang code